README.md

```diff
@@ -199,6 +238,30 @@ The `-k` option is added here as a self-signed cert, and this allows cURL to
 warn and proceed. For production, use an HTTPS strategy signed by a certificate
 authority instead of a self signed cert.
 
+## Testing
+
+To run the end-to-end tests:
```

Comment:
Maybe we should mention this will run against the paid API.

Reply:
Good call, I added a note that these will count as billed assessments in the readme